ansible-advancedlinux/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Deploy N Linux VMs and set Ansible up to efficiently manage all the VMs",
```

Comment:
Suggestion: For the Azure.com index table, it might be better to have the name "Setup Ansible to manage N Linux VMs" so that it fits in the column width